packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -131,4 +137,16 @@ const RenderBackgroundScenario = ({ background }: ScenarioProps) => {
   );
 };
 
+export const ExamplesWithoutTooltip = () => {
+  const sizes: IconSize[] = ['xs', 'sm', 'md', 'lg', 'xl'];
+
+  return (
+    <HorizontalGroup justify="center">
```

Comment:
Maybe a little text here to explain that in this case we are using the aria-label? Otherwise this is a strange storybook story 😄

Reply:
That's true. I assumed that people would read the docs but I know this sounds silly. 😅